run_test.py

```diff
@@ -31,12 +31,11 @@
         script = './test.sh'
 
     elif args.test == 'py3':
-        # TODO(unno): Fix this line after protobuf 3 is released
         conf = {
             'base': 'ubuntu14_py3',
             'cuda': 'cuda70',
             'cudnn': 'cudnn4',
-            'requires': ['setuptools', 'cython', 'numpy<1.11', 'protobuf==3.0.0b3'],
+            'requires': ['setuptools', 'cython', 'numpy<1.11', 'protobuf<3'],
```

Comment:
I would like you *why* you use old protobuf.
I guess it is because we would like to use every version of third party libraries and pythons at least once, as we cannot test all combinations of them and have to restrict the combinations to be tested. That is what I suppose to be written in the comment. But I come to think we should write such things in README, instead of inline comment.